.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+If you added a new package make sure to check for these items:
```

Comment:
Just a reminder for someone to not forget some steps when adding new packages.

Reply:
@mislavmiocevic I think it would make sense to rephrase this as:

> If you are adding a new package:

and at the first point, change it to:

> Add a new connection to the `graph TD` code block section in the `README.md` file of the root project, to update the diagram created with [Mermaid](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/)

(note these changes: "__to__ the `graph TD`" and "__with__ Mermaid")

And please remove the _dots_ at the ends of the points

Reply:
Done

https://github.com/infinum/js-linters/pull/66/commits/d4b0aea594f8d3c582c3aa3a5f463c05493a9d16